### PR TITLE
chore: misc fmt

### PR DESCRIPTION
**Pull request by mattsse**
reorder
add debug

### CODE REVIEW

File: crates/fmt/src/shadow/mod.rs
Comment (DaniPopes) [inline crates/fmt/src/shadow/mod.rs:1]: empty?

File: crates/fmt/src/formatter.rs
Comment (DaniPopes) [inline crates/fmt/src/formatter.rs:1]: Is this an IDE thing?

**Review by DaniPopes — CHANGES_REQUESTED** [review]

File: crates/fmt/src/shadow/mod.rs
Comment (mattsse) [reply crates/fmt/src/shadow/mod.rs:1]: was auto added...
removed

File: crates/fmt/src/formatter.rs
Comment (mattsse) [reply crates/fmt/src/formatter.rs:1]: yeah, rustrover shows a warning if trait impl functions have a different order than the trait itself

Comment (DaniPopes) [reply crates/fmt/src/formatter.rs:1]: nice

**Pull request merged**